notebooks/post_DL3_analysis.ipynb

```diff
@@ -1,36 +1,58 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            fits.BinTableHDU(flux_points.to_table(), name="SED"),
</pre><p>One can also do directly:</p>

<pre class="ql-syntax" spellcheck="false">flux_points.write("filename_flux_points.fits", sed_type="e2dnde", format="gadf-sed", overwrite=True)
lc_1d.write("filename_lightcurve.fits", sed_type="flux", format="lightcurve")
</pre><p>to store the SED and LC flux points separatelly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>